.travis.yml

```diff
@@ -22,6 +26,12 @@ matrix:
       env: DJANGO=2.2 DB=postgres
     - python: "2.7"
       env: DJANGO=2.2 DB=sqlite
+    - python: "2.7"
+      env: DJANGO=3.0b1 DB=mysql
```

Comment:
We also need on line 67
```
if [[ "$DB" == "mysql" && "$DJANGO" == "3.0b1" ]]; then pip install mysqlclient; fi
```
Or some way to get mysqlclient installed when we need it.